src/bananas.js

```diff
@@ -0,0 +1,180 @@
+define([
+    'vellum/xpath',
+], function(
+    xpath
+) {
+    var OUTSIDE_BANANA = 0,
+        INSIDE_BANANA = 1,
+        DELIMITER = "🍌",
```

Comment:
I got overwhelmed by :banana:s last night when I originally tried to review this PR. They are cute, but I think there are serious disadvantages to using them as a delimiter:

- it's a [4-byte character](https://mothereff.in/byte-counter#%F0%9F%8D%8C)
- there's no easy way to type it on a standard keyboard
- using an obscure character doesn't mean you don't need to escape it properly (the data we're dealing with may contain it and we should handle that situation properly)

I personally don't like this delimited-bubbles thing at all (hashtag expression parsing just got even more complex, and I'm not convinced we need it), but if I had to vote I would say back-tick (`` ` ``) would be a better delimiter.